pkgs/servers/nosql/mongodb/mongodb.nix

```diff
@@ -97,9 +105,10 @@ in stdenv.mkDerivation rec {
   meta = {
     description = "A scalable, high-performance, open source NoSQL database";
     homepage = http://www.mongodb.org;
-    license = licenses.agpl3;
+    license = licenses.sspl;
```

Comment:
Won’t this mean that AGPL versions of MongoDB are marked as non-free?

Reply:
Good question, according to the [MongoDB README](https://github.com/mongodb/mongo/blob/b73a7d6ab9d38382a6c2f1fd0b5c8369d54d8b95/README#L89):

```
LICENSE

  MongoDB is free and the source is available. Versions released prior to
  October 16, 2018 are published under the AGPL. All versions released after
  October 16, 2018, including patch fixes for prior versions, are published
  under the Server Side Public License (SSPL) v1. See individual files for
  details.
```

Versions [3.4.22](https://docs.mongodb.com/manual/release-notes/3.4/#aug-6-2019), [3.6.13](https://docs.mongodb.com/manual/release-notes/3.6/#jun-10-2019) and [4.0.11](https://docs.mongodb.com/manual/release-notes/4.0/#jul-26-2019) were all released after October 16, 2018; so they all have to be marked as SSPL.

Reply:
In that case, I’d be inclined to either not offer patch versions after that date, or to at least continue to offer the last free version.